Modules/Core/Common/include/itkImageBase.hxx

```diff
@@ -474,10 +475,7 @@ void
 ImageBase< VImageDimension >
 ::SetRequestedRegion(const RegionType & region)
 {
-  if ( m_RequestedRegion != region )
-    {
-    m_RequestedRegion = region;
```

Comment:
It is common to have to update the MTime of an object when an IVAR is modified to ensure the pipeline will be updated after the change. As the Requested region is part of the pipeline state, the MTime does not need to be modified if the IVAR changes. See SetLargestPossibleRegion for the difference.

Reply:
Thanks for your explanation, @blowekamp  I'm glad to hear that it's not a bug that `ImageBase::SetRequestedRegion` does not do `this->Modified()`.